ee/koor_release_process.md

```diff
@@ -17,7 +17,7 @@ We follow the release process of Rook upstream and also sync our latest releases
   # set to the new release
   tag_name=<release version> # e.g., v1.10.9
   git tag -a "$tag_name" -m "$tag_name release tag"
-  git push upstream "$tag_name"
+  git push upstream "$tag_name" && git push upstream release-1.10
```

Comment:
Could you add a comment to what the `release-1.10` should be as an example see line 18.